__tests__/core/runner.test.ts

```diff
@@ -809,4 +809,18 @@ describe('runner', () => {
     expect(monitors.length).toBe(1);
     expect(monitors[0].config.name).toBe('j1');
   });
+
+  it('runner - build monitors filtered through "tags" added updating monitor config', async () => {
```

Comment:
Instead of having this test in runner, it should be on the push command as monitor.use only touches the pushed monitors and never deals with runner parameters.

Reply:
thank you for the context 🚀 . I will have a look at this

Reply:
done!